sdk/core/performance-stress/src/program.cpp

```diff
@@ -89,7 +82,8 @@ inline void PrintOptions(
     {
       try
       {
-        optionsJs[option.Name] = parsedArgs[option.Name].as<std::string>();
+        optionsJs[option.Name]
```

Comment:
Nit, optionsJs isn't clear on what it is.  I have to go find it to realize Js was short for json.